src/actions/play-pause.action.ts

```diff
@@ -83,8 +87,8 @@ export class PlayPauseAction extends DefaultAction<PlayPauseAction> {
             this.plugin.setTitle(this.currentTitle, context);
         }
 
-        if (this.playing !== data.player.isPaused) {
-            this.playing = data.player.isPaused;
+        if (this.playing === data.player.isPaused) {
+            this.playing = !data.player.isPaused;
```

Comment:
What exactly does this accomplish / change? You just reversed the operators.

Reply:
When the player "is paused" that means that the track is playing. This fixed a bug where the play/pause icons were flipped. When the song is playing the streamdeck button should display a "pause" button indicating that pressing the button will pause the music. When the song is paused it should display a "play" button indicating that pressing the button will play the music. (This is how the play/pause button behaves within the YTMusic Desktop app itself.)